How to get Library / Dev Env Values in custom code (#282)
FlutterFlow/flutterflow-documentation: docs/ff-concepts/adding-customization/common-examples.md

- These variables make it easy to integrate Firebase Auth data into custom functionality, enhancing the user experience.

### Get Dev Environment Values in Custom Code

Similar to the `FFLibraryValues` class, if you are using **[Dev Environments](../../testing-deployment-publishing/development-environments/development-environments.md)** in your FlutterFlow project, a new class called `FFDevEnvironmentValues` will be created. This class can also be accessed from custom code if needed. It is generated based on the environment selected by the user at the time of code generation.

To access any Dev Environment values in custom code, simply use:

```js
Future getWebhookId() async {
  // Add your function code here!
  return FFDevEnvironmentValues().webhookId;
}
```

### Access Library Components in Custom Code

When using a library dependency in your project, you can also access its components, such as Library App State, Library Values, and Library Widgets, in the user project's custom code. Here are a few examples:

#### Get Library Values 

> Similar to the `FFAppState` class, FlutterFlow generates an `FFLibraryValues` class, which is also an abstract class. This class provides access to the **[Library Values](../../resources/projects/libraries.md#library-values)** if the project is a Library.

Reviewer: ```suggestion
Similar to `FFAppState`, FlutterFlow generates an abstract `FFLibraryValues` class for library projects, which provides direct access to **[Library Values](../../resources/projects/libraries.md#library-values)**.
```

Author: Taking the other changes, but my bad, i meant singleton class and not abstract class.
